core/src/main/java/zipkin/reporter/AsyncReporter.java

```diff
@@ -156,7 +156,9 @@ public Builder queuedMaxBytes(int queuedMaxBytes) {
             for (byte[] next : consumer.drain()) result.pending.offer(next);
             result.close.countDown();
           }
-        }, "AsyncReporter(" + sender + ")").start();
+        }, "AsyncReporter(" + sender + ")");
+        flushThread.setDaemon(true); //Let applications terminate even if thread is not properly closed.
```

Comment:
there's no need for the trailing comment here – we know what daemon thread implies (or can just the apidoc on it)